### PR TITLE
Expand Ansible inventory by default on the new host page

### PR DESCRIPTION
**Pull request by nofaralfasi**
Expand Ansible inventory by default to show Ansible roles.

### CODE REVIEW

File: app/helpers/foreman_ansible/ansible_hostgroups_helper.rb [hunk 1]
```diff
@@ -3,13 +3,9 @@
...
     def ansible_hostgroups_actions(hostgroup)
-      play_roles = if hostgroup.all_ansible_roles.empty?
-                     { action: (link_to _('Run all Ansible roles'), 'javascript:void(0);', disabled: true, title: 'No Roles assigned'), priority: 31 }
-                   else
-                     { action: display_link_if_authorized(_('Run all Ansible roles'), hash_for_play_roles_hostgroup_path(id: hostgroup), 'data-no-turbolink': true, title: _('Run all Ansible roles on hosts belonging to this host group')), priority: 31 }
-                   end
+      return [] if hostgroup.all_ansible_roles.empty? || !User.current.can?(:create_job_invocations)
 
-      [play_roles] if User.current.can?(:create_job_invocations)
+      [{ action: display_link_if_authorized(_('Run all Ansible roles'), hash_for_play_roles_hostgroup_path(id: hostgroup), 'data-no-turbolink': true, title: _('Run all Ansible roles on hosts belonging to this host group')), priority: 31 }]
```
Comment (stejskalleos) [inline app/helpers/foreman_ansible/ansible_hostgroups_helper.rb:8]: Isn't this from the https://github.com/theforeman/foreman_ansible/pull/554?

Comment (nofaralfasi) [reply app/helpers/foreman_ansible/ansible_hostgroups_helper.rb:8]: Yes, my bad. Good thing you noticed that.

**Comment by nofaralfasi** [comment]
A screenshot demonstrating how the new Inventory tab will expand by default: 
image

File: webpack/components/ReportJsonViewer.js [hunk 2]
```diff
@@ -10,7 +10,17 @@ const theme = {
 
 const ReportJsonViewer = ({ data }) => (
   <div className="report-json-viewer">
-    <JSONTree data={data} hideRoot theme={theme} />
+    <JSONTree
+      data={data}
+      hideRoot
+      theme={theme}
+      shouldExpandNode={(keyPath, myData, level) =>
```
Comment (stejskalleos) [inline webpack/components/ReportJsonViewer.js:17]: +1 for the catch with `level === 3`.

Code can be slightly simplified:
```js
      shouldExpandNode={(keyPath, _myData, level) =>
        keyPath[0] === '_meta' ||
        keyPath[0] === 'hostvars' ||
        level === 3 ||
        keyPath[0] === 'foreman_ansible_roles'
      }
```
Chances that there will be `_meta`, `hostvars` or `foreman_ansible_roles` on other levels are pretty slim,
so we can skip the level condition IMHO. What do you think?

The underscore in `_myData` tells developer that this variable is not used in the function, it's just there doing nothing :D

**Review by stejskalleos — APPROVED** [review]
:green_apple: **LGTM**, works as expected!

**Pull request merged**